### DISCUSSION TITLE
Build error with Next/Image

### DISCUSSION THREAD

**Discussion by tiavina-mika** [vercel/next.js#21480] [Help] [answered]
**What version of Next.js are you using?**

10.0.2

**What version of Node.js are you using?**

14.15.4

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows 10

**How are you deploying your application?**

Other platform

**Describe the Bug**

I'm using Next/Image. I got the error below when I'build my app with next build & next export:

Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
Possible solutions:
  - Use `next start`, which starts the Image Optimization API.
  - Use Vercel to deploy, which supports Image Optimization.
  - Configure a third-party loader in `next.config.js`.

**Expected Behavior**

.

**To Reproduce**

.

![image](https://user-images.githubusercontent.com/42656064/105610220-9bdcaf00-5dbf-11eb-9cad-c7ac94ea0d45.png)

**Comment by jamesmosier** [comment, marked as answer]
Hi @tiavina-mika. As the error mentions, `next/image` and `next export` are not compatible. See [err.sh/next.js/export-image-api](https://err.sh/next.js/export-image-api) for more info, but the gist is that `next/image` is a server side optimization at runtime of your application. When you use `next export` this server side "API" is not available because your Next.js application is just static HTML/CSS/JS without a Node server side environment which can handle the image processing.

To resolve this you would need to just run `next build` and then `next start` to start up your application.

**Reply by uxduck** [reply]
Does this mean it won't work for **static** hosting? This forces people to run a proper hosted server, since they need to run `next start`? Or of course, use serverless (e.g. Cloudflare workers) to run the application. But running `next start` is not instantaneous.

**Reply by uxduck** [reply]
I know that it works fine in the vercel ecosystem though :), but I like to not be locked in :)

**Comment by tiavina-mika** [comment]
@jamesmosier , but I need to serve the static html file from next export.
So I should not use the image optimization for that?

**Reply by jamesmosier** [reply]
If you need to use `next export` then you cannot use `next/image`.

**Reply by TileRunner** [reply]
I have the same issue. I use npm run build and npm run export. I want to use an internal image from my public folder. How do I do it without getting export errors?

**Reply by karltaylor** [reply]
I believe you will have to use the standard `<img />` component.

**Reply by aysebilgen** [reply]
I had the same problem. I could not find any solution. I remove all next/image in all my codes and use basic-standard <img/> and i can build and export my project for static hosting.
